foreshadow/core/preparerstep.py

```diff
@@ -427,3 +515,17 @@ def inverse_transform(self, X, *args, **kwargs):
         """
         self.check_process(X)
         return self._parallel_process.inverse_transform(X, *args, **kwargs)
+
+    @staticmethod
+    def _preparer_params():
```

Comment:
Is this the patch to for the sklearn get_params?

Reply:
Yes, for the __repr__ in this case

Reply:
Issue created #117